### DISCUSSION TITLE
Breaking changes in the "main" branch by using CSS variables?

### DISCUSSION THREAD

**Discussion by okadurin** [twbs/bootstrap#36037] [Q&A]
Let's say I'm using bootstrap@5.1.3 and there is a piece of code like this:
```
@import "~bootstrap/scss/bootstrap";

.alert.alert-warning {
  border-width: $alert-border-radius * 2;  
}
```
It works on 5.1.3 because `$alert-border-radius` is a SCSS variable: https://github.com/twbs/bootstrap/blob/1a6fdfae6be09b09eaced8f0e442ca6f7680a61e/scss/_variables.scss#L1451

Then when switching to the `main` branch the piece of code above is going to fail with error:

```
SassError: Undefined operation "var(--bs-border-radius) * 2"
```
That is because the same variable now contains a CSS variable value: https://github.com/twbs/bootstrap/blob/b165f35f7808b59f4e09ec518caa441e30830fbb/scss/_variables.scss#L1432
----
To mitigate this I'd need to update my code to use `calc`:
`border-width: calc(#{$alert-border-radius} * 2);`  

The question is: is it expected? I understand that this is just a `main` branch and not something which has been released. But I'd like to understand how it's going to be in the future. Will migration be required?

**Comment by mdo** [comment]
Yeah, I might need to roll this back a bit and not use the CSS variables in the main Sass variables until v6. Let me think on this a bit more.
